src/test/resources/Hotels/HotelsCheckIn.feature

```diff
@@ -0,0 +1,24 @@
+Feature: verification of sorting of HotelsPrice
+
+  Scenario: Verify that on sorting the hotel price the lowest hotel price option is less than or equal to 100
```

Comment:
Before scenario name you should have some tag like @booking. These tags will help you to run specified test/s through test runner.